package.json

```diff
@@ -12,35 +12,35 @@
   },
   "dependencies": {
     "@apollo/client": "^3.11.1",
-    "@expo/metro-config": "0.18.9",
-    "@react-native-community/datetimepicker": "^8.2.0",
+    "@expo/metro-config": "~0.18.11",
+    "@react-native-community/datetimepicker": "8.0.1",
```

Comment:
why is this being downgraded?

Reply:
Running `expo install --fix` results in this change automatically.